src/entities/card/index.tsx

```diff
@@ -0,0 +1,89 @@
+import { FC } from 'react';
+import { useState } from 'react';
+import { Box } from '@mui/material';
+import { Paper } from '@mui/material';
```

Comment:
В импортах можно объединять импорты из одной библиотеки

```suggestion
import { Box, Paper } from '@mui/material';
```